ui/lib/ldap/addon/components/page/roles.hbs

```diff
@@ -43,46 +43,59 @@
 {{else}}
   <div class="has-bottom-margin-s">
     {{#each @roles as |role|}}
-      <ListItem @linkPrefix={{this.mountPoint}} @linkParams={{array "roles.role.details" role.type role.name}} as |Item|>
+      <ListItem @linkPrefix={{this.mountPoint}} @linkParams={{this.linkParams role}} as |Item|>
         <Item.content>
           <Icon @name="user" />
-          <span data-test-role={{role.name}}>{{role.name}}</span>
+          <span data-test-role="{{role.type}} {{role.name}}">{{role.name}}</span>
           <Hds::Badge @text={{role.type}} data-test-role-type-badge={{role.name}} />
         </Item.content>
         <Item.menu>
           <Hds::Dropdown @isInline={{true}} @listPosition="bottom-right" as |dd|>
-            <dd.ToggleIcon @icon="more-horizontal" @text="More options" @hasChevron={{false}} data-test-popup-menu-trigger />
-            {{#if role.canEdit}}
+            <dd.ToggleIcon
+              @icon="more-horizontal"
+              @text="More options"
+              @hasChevron={{false}}
+              data-test-popup-menu-trigger="{{role.type}} {{role.name}}"
+            />
+            {{#if (this.isHierarchical role.name)}}
```

Comment:
🤔 Does this need `fn` wrapped around, so it's clear you're passing role.name into `this.isHierarchical()`

Reply:
This is actually a template helper so it has slightly different syntax 😄  [Docs](https://guides.emberjs.com/release/components/helper-functions/#toc_local-helper-functions)

<img width="841" alt="Screenshot 2024-11-05 at 4 19 15 PM" src="https://github.com/user-attachments/assets/3a159065-874f-4d82-8b4c-68c2f6fa1eb7">

Reply:
TIL, thank you!